src/dsp_tools/commands/xml_validate/xml_validate.py

```diff
@@ -42,9 +43,14 @@ def xml_validate(filepath: Path, api_url: str, dev_route: bool) -> bool:  # noqa
     onto_con = OntologyConnection(api_url, shortcode)
     ontologies, shapes = _get_shacl(onto_con)
     data_graph = data_rdf.make_graph()
+    generic_filepath = Path()
+    if save_graphs:
+        generic_filepath = _save_graph(filepath, ontologies, shapes, data_graph)
     # data_graph += ontologies
     val = ShaclValidator(api_url)
     report = _validate(val, shapes, data_graph)
+    if save_graphs:
```

Comment:
I am doing this separately in case the validation fails. We at leas have the other files